content/en/docs/how-tos/cluster-claim.md

```diff
@@ -3,62 +3,223 @@ title: "Creating a Cluster Pool"
+1. `metadata.labels`: These labels will be used by jobs to specify what cluster they want to obtain for their testing.
+2. `spec.baseDomain`: A base domain for all clusters created in this pool. You usually create this domain yourself when
+   configuring the cloud platform account (see [prerequisites](#prerequisites)).
+3. `spec.imageSetRef`: A reference to a `ClusterImageSet` in the cluster that determines the exact version of clusters
+   created in the pool. `ClusterImageSets` are cluster-scoped resources and their manifests are present
+   in [`clusters/hive/pools`](https://github.com/openshift/release/tree/master/clusters/hive/pools) directory. Either
+   select one of the already available, or create a new one. DPTP maintains a set of `ClusterImageSets` that are
+   regularly bumped to most recent released OCP versions.
+4. `spec.installConfigSecretTemplateRef`: a reference to a `Secret` that serves as an installation config template. See
+   the [below section](#create-a-manifest-for-your-install-config-template-secret) for more information.
+5. `spec.platform.$CLOUD.credentialsSecretRef`: A reference to the secret you created in
+   the [Prepare your cloud platform credentials](#prepare-your-cloud-platform-credentials) section.
+6. `pullSecretRef.name`: Must be kept `pull-secret`. OpenShift CI will populate your namespace with this secret that
+   contains all pull secrets necessary to install an OCP cluster.
 
-If any manifest contains sensitive information, e.g., `secret/hive-aws-credentials`, see [how to add a secret to CI](/docs/how-tos/adding-a-new-secret-to-ci/). Make sure to set the key `secretsync/target-clusters: "hive"` for the secret to ensure it is synced only to the cluster that needs it.
+#### Sizing your cluster pool
+
+Hive maintains the number of clusters in the pool as specified by its `size`. A provisioned cluster will
+be [hibernating](https://github.com/openshift/hive/blob/master/docs/hibernating-clusters.md) after staying idled for
+some time and can be woken up if a job claims it. Hive removes it from the pool once it is claimed and creates a new
+cluster to maintain the pool's `size`. The cluster will be destroyed after 4 hours since it is claimed. If several jobs
+claim a cluster simultaneously, Hive will feed all of them up until the number of living clusters reaches the
+pool's `maxSize`.
+
+All clusters before destruction consume resources in the cloud account, so your `maxSize` should be set according to
+your cloud platform limits and quotas and presence of other cluster pools or other resource consumption in your cloud
+platform account.
```

Comment:
we asked for a max in flight knob to allow limiting the number of installs and deprovisions happening at once to limit aws throttling - unclear if it's worth it to mention here

Reply:
I'm leaning towards not: rate limits are transparent to the user, the failures happen in background and are retried. Once the knob is exposed, we can add it.